docs/dev/check_references.md

```diff
@@ -110,7 +110,7 @@ You can find the complete list of mandatory and optional attributes for the `man
 | `tile[media[media_type]]`                            | String or Enum              | Mandatory                                 | The type of media this is. Allowed values are `image` and `video`.                                                                                                                                                                          |
 | `tile[media[caption]]`                               | String                      | Mandatory                                 | The caption for the image.                                                                                                                                                                                                                  |
 | `tile[media[image_url]]`                             | String                      | Mandatory                                 | The relative path to this image in relation to this manifest file.                                                                                                                                                                          |
-| `tile[classifier_tags]`                              | Array of String             | Mandatory, can be `[ ]`                   | Some classifier information about this app. This includes information such as `supported_os` and `available_offerings`.                                                                                                                     |
+| `tile[classifier_tags]`                              | Array of String             | Mandatory, can be `[ ]`                   | This includes information such as `categories`, `submmited/queried data types`, `supported_os` and `available_offerings`.                                                                                                                     |
```

Comment:
```suggestion
| `tile[classifier_tags]`                              | Array of String             | Mandatory, can be `[ ]`                   | Classifier information about this app such as `categories`, `submmited/queried data types`, `supported_os`, and `available_offerings`.                                                                                                                     |
```
There is more than one good way of phrasing this information, so feel free to edit my wording. I didn't like the "This includes..." wording because it is different from the grammar of the other entries in the table. And I added in the Oxford comma.